cobalt/browser/metrics/cobalt_metrics_service_client.h

```diff
@@ -72,6 +85,9 @@ class CobaltMetricsServiceClient : public metrics::MetricsServiceClient {
       const ::metrics::MetricsLogUploader::UploadCallback& on_upload_complete)
       override;
   base::TimeDelta GetStandardUploadInterval() override;
+  void SetUploadInterval(base::TimeDelta interval);
```

Comment:
nit: Only overrides are put in a contiguous block, so pls add here a blank line above this guy (and prob move the comment in l.91 to here, next to the overrides).

Reply:
Done